utils/fo-installdeps

```diff
@@ -178,23 +178,23 @@ if [ $RUNTIME ]; then
          if [ -z "$POSTGRE" ]; then  ## if postgresql-server-dev is installed
            case "$CODENAME" in
               squeeze)
-                 apt-get $YesOpt install postgresql-8.4;;
+                 apt-get $YesOpt install postgresql-8.4 libapache2-mod-php5 php5 php5-pgsql php5-cli php5-curl;;
               wheezy)
-                 apt-get $YesOpt install postgresql-9.1;;
-              sid)
-                 apt-get $YesOpt install postgresql-8.4;;
+                 apt-get $YesOpt install postgresql-9.1 libapache2-mod-php5 php5 php5-pgsql php5-cli php5-curl;;
               lucid|natty)
-                 apt-get $YesOpt install postgresql-8.4;;
+                 apt-get $YesOpt install postgresql-8.4 libapache2-mod-php5 php5 php5-pgsql php5-cli php5-curl;;
               isadora)
-                 apt-get $YesOpt install postgresql-8.4;;
+                 apt-get $YesOpt install postgresql-8.4 libapache2-mod-php5 php5 php5-pgsql php5-cli php5-curl;;
               precise|oneiric|quantal|raring|saucy)
-                 apt-get $YesOpt install postgresql-9.1;;
+                 apt-get $YesOpt install postgresql-9.1 libapache2-mod-php5 php5 php5-pgsql php5-cli php5-curl;;
               trusty)
-                 apt-get $YesOpt install postgresql-9.3;;
+                 apt-get $YesOpt install postgresql-9.3 libapache2-mod-php5 php5 php5-pgsql php5-cli php5-curl;;
               jessie)
-                 apt-get $YesOpt install postgresql-9.4;;
-              xenial)
-                 apt-get $YesOpt install postgresql-9.5;;
+                 apt-get $YesOpt install postgresql-9.4 libapache2-mod-php5 php5 php5-pgsql php5-cli php5-curl;;
+              xenial|yakkety)
+                 apt-get $YesOpt install postgresql-9.5 libapache2-mod-php5 php5 php5-pgsql php5-cli php5-curl;;
```

Comment:
Drop the '5' from php packages?

Reply:
keestux - to be clear, when you say drop support for php5, what you are really suggesting is to drop support for: squeeze, wheezy, lucid, natty, isadora, precise, oneiric, quantal, raring, saucy, trusty, jessie, xenial, yakkety

if you are only suggesting to drop the number '5' and require the meta-packages php, php-pgsql, php-cli, php-curl, etc. instead of php,5 php5-pgsql, php5-cli, php5-curl - that has already been suggested in this PR

Reply:
Nevermind. I just got confused by Github which showed commit a4af5cc, but which is not present in the fresh clone that I made.